### ISSUE TITLE
Collector Drop Off Repair Speed

### ISSUE THREAD

**Issue by cloaked2222** [S5SS/dual-command#65]
CloakedToday at 10:51 PM
Btw, when colls are damaged, ru drop offs back-up for miles to SLOWLY heal colls. Should their heal rate go up a bit?
FearToday at 10:52 PM
lol i saw that
in your match
you think hw2 has it bad
hw1 refs have 1 slot
and the colls have more hp to heal
CloakedToday at 10:52 PM
Relic prolly set it for lv0 health hw2 colls. but hw1 and fully upgraded hw2 colls heal super slow.
doubt gearbox ever changed, so prolly something to consider improving.
FearToday at 10:52 PM
yeah
CloakedToday at 10:53 PM
Currently you have to 'dock' them to heal the fast, and then remember to tell them to harvest again....
FearToday at 10:53 PM
at the least you should be able to undock the ones hogging the drop
CloakedToday at 10:53 PM
something
maybe my G1 torp was on to something, clogging up the drop off bettter than killing colls? :p
FearToday at 10:53 PM
at the least you should be able to undock the ones hogging the drop

**Comment by Novaras** [comment]
could this have been affected by the nerfs done to hw2 docking replenish speed?

**Comment by cloaked2222** [comment]
No, you can set a different heal rate for each ship class. For collectors this has been 35 (looking at hiig MS) since the begging of time. The code below applies to docking inside the MS, and probably colls dropping off RU too.
http://hw2wiki.net/wiki.hw2.info/FunctionaddAbility_show_comments=1.html

2.205:
addAbility(NewShipType,"ShipHold",1,250,200,"rallypoint","Fighter, Corvette, Utility",35,{SuperCap="2000"},{Frigate="800"},{Fighter="36"},{Corvette="75"});

2.3b9:
addAbility(NewShipType,"ShipHold",1,250,200,"rallypoint","Fighter, Corvette, Utility",35,{SuperCap="2000"},{Frigate="800"},{Fighter="3"},{Corvette="32"});

**Comment by S5SS** [comment]
I think most people would agree that it should be worse to lose a collector than have one clog up a drop off for up to 35s so Id be fine with a speed increase.
